controllers/informationCenter/InformationCenterHandler.inc.php

```diff
@@ -27,7 +27,7 @@
 	function __construct() {
 		parent::__construct();
 		$this->addRoleAssignment(
-			array(ROLE_ID_SUB_EDITOR, ROLE_ID_MANAGER, ROLE_ID_ASSISTANT),
+			array(ROLE_ID_SUB_EDITOR, ROLE_ID_MANAGER),
```

Comment:
(I think this dates from when the metadata view was part of the information centre)

Reply:
Not sure what you're indicating here. Do you think the role assignment should change?

Reply:
No, it's OK -- it looks to me like you've clarified `ROLE_ID_ASSISTANT` access to the information centre, where there may have been a leftover grant for assistants to the submission information center. Just mentioning why things might have been that way in case it helps with the history.